src/afl-showmap.c

```diff
@@ -154,7 +154,11 @@ static u32 write_results_to_file(afl_forkserver_t *fsrv, u8 *outfile) {
 
   if (!strncmp(outfile, "/dev/", 5)) {
 
+#if !defined(__ANDROID__)
     fd = open(outfile, O_WRONLY, 0600);
+#else
```

Comment:
Anyone knows why this has the permissions/mode is in there?
On normal Linux it is only for O_CREAT operations and otherwise ignored. and this has no O_CREAT.
@hexcoder-  - does *BSD need that?

Reply:
seemingly not

Reply:
BSD needs permissions only for O_CREAT, too. Does not make sense otherwise IMO.